### ISSUE TITLE
用語集「コントラスト比」の注記4について

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1396]
4つ目の注記、この訳は正確か？（failureを問題があるというように訳してよいか）

https://waic.jp/docs/WCAG21/#dfn-contrast-ratio

> 背景色は、テキストが通常の使用においてレンダリングされるときに背景となるコンテンツの色として指定された色である。テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、問題がある。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も問題がある。

https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio

> Background color is the specified color of content over which the text is to be rendered in normal usage. It is a failure if no background color is specified when the text color is specified, because the user's default background color is unknown and cannot be evaluated for sufficient contrast. For the same reason, it is a failure if no text color is specified when a background color is specified.

**Comment by caztcha** [comment]
達成方法集の F (失敗例) の記述に倣って、failure を「達成基準の失敗」と訳すのは、いかがでしょう？

> 背景色は、テキストが通常の使用においてレンダリングされるときに背景となるコンテンツの色として指定された色である。テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、**達成基準の失敗となる**。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も**達成基準の失敗となる**。

**Comment by bakera** [comment]
要するにF24の失敗例に該当するという話ですよね。

https://waic.jp/docs/WCAG21/Techniques/failures/F24
https://www.w3.org/WAI/WCAG21/Techniques/failures/F24

F系の訳では failure = 失敗例としていて、「達成基準の」などは補っていません。
これもそのまま「失敗例となる」としてしまっても良いのかもしれません。

**Comment by caztcha** [comment]
@bakera ありがとうございます。
なるほど、failure を単に「失敗」とするとしっくりこないなと思ってこねくり回してしまいましたが、「失敗例」とすれば、収まりよい感じがしますね。

> 背景色は、テキストが通常の使用においてレンダリングされるときに背景となるコンテンツの色として指定された色である。テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、**失敗例となる**。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も**失敗例となる**。

これでプルリクエスト作成します。

**Comment by caztcha** [comment]
PR #1405 で対応中です。
